pipelines/environments/promptflow-runtime/context/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mcr.microsoft.com/azureml/promptflow/promptflow-runtime:{{latest-image-tag}}
```

Comment:
We don't create registry environments just to point to other images. Please include the actual Docker build context of this environment here.

Reply:
base image has out of SLA vulnerability and doesn't follow convention for tagging

"VulnerabilityId": 992484,
"AssetId": sha256:778dc5423f681247d485658271196c44d976d978cb52fe78d077302196ef428a,
"RunId": 2023-05-16T13:51:22.3306902Z,
"ImageName": azuremlmcr.azurecr.io/public/azureml/promptflow/promptflow-runtime:latest,
"IsActionable": true,
"VulnerabilityName": GO (Go) Security Update for google.golang.org/protobuf (GHSA-hw7c-3rfg-p46j),
"DueDate": 2023-04-15T00:00:00Z,

Reply:
For the tagging issue, we can update our pipeline to not push the latest image to `azuremlmcr`.

And for the golang issue, we will try to fix that.

Reply:
have updated to fix the golang vulnerability issue, and the image tag is pushed with both YYYYMMDD.vX and latest.

Reply:
and for the issue Louie pointed out, let's treat it as a short-term solution, will include related build context after the PuP of related SDK - resolving this discussion.